apps/extension/src/ui/hooks/useCurrency.tsx

```diff
@@ -0,0 +1,19 @@
+import { selectableCurrenciesState, selectedCurrencyState } from "@ui/atoms"
+import { useRecoilCallback } from "recoil"
+
+export const useToggleCurrency = () =>
```

Comment:
Let's give the file the same name as the hook, since `useCurrency` is a bit general and doesn't describe the function of the hook.

Reply:
I'm following naming pattern used by others files that exports multiple hooks i.e `useSettings`, `useFeatures`. Right now, this file only exports a single hook, but naming it `useCurrency` allows us to add more hooks later without changing the file name.

Reply:
There are some exceptions, but the main pattern in the codebase by far is to have the file name the same as the main exported hook function.

Reply:
With the addition of `useSelectedCurrency` this file name now make sense